cluster/common.sh

```diff
@@ -933,3 +933,19 @@ function parse-master-env() {
   KUBELET_CERT_BASE64=$(get-env-val "${master_env}" "KUBELET_CERT")
   KUBELET_KEY_BASE64=$(get-env-val "${master_env}" "KUBELET_KEY")
 }
+
+# Writes the number of master replicas ($1) to the cluster config map.
+function update-cluster-config-map() {
+  local num_replicas="${1}"
+  echo "Updating cluster config map, the new number of replicas: ${num_replicas}"
+  local kubectl="${KUBE_ROOT}/cluster/kubectl.sh"
+  "${kubectl}" replace --force -f - <<EOF
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: apiserver-config
```

Comment:
Is this really apiserver-specific, or is it multi-component?  Somethign we did for the kubelet config was to think of the config as a per-node config, and prefix keys with the component.  So in this case it might be named "cluster-config", and the key named "kube-apiserver.config"

@mtaufen